flexmeasures/api/v3_0/sensors.py

```diff
@@ -273,6 +273,18 @@ def index(
     @route("<id>/data/upload", methods=["POST"])
     @path_and_files(SensorDataFileSchema)
     def upload_data(self, data: list[tb.BeliefsDataFrame], **kwargs):
+        """
+        Upload a file containing sensor data to be stored in FlexMeasures.
+
+        The payload should be a file containing sensor data in CSV format.
```

Comment:
This helpful text should also be displayed when hovering an info icon in the firm, so users understand what file to add.

Reply:
The other pot endpoint has this helpful text which we could repeat here:

"
The unit has to be convertible to the sensor's unit.
The resolution of the data has to match the sensor's required resolution, but FlexMeasures will attempt to upsample lower resolutions.
The list of values may include null values.
"